pkg/config/config.go

```diff
@@ -412,10 +412,11 @@ type (
 	}
 
 	edgeRuntime struct {
-		Enabled       bool          `toml:"enabled"`
-		Image         string        `toml:"-"`
-		Policy        RequestPolicy `toml:"policy"`
-		InspectorPort uint16        `toml:"inspector_port"`
+		Enabled         bool          `toml:"enabled"`
+		Image           string        `toml:"-"`
+		Policy          RequestPolicy `toml:"policy"`
+		InspectorPort   uint16        `toml:"inspector_port"`
+		DefaultLanguage string        `toml:"default_language"`
```

Comment:
Does this mean we can't support js and ts at the same time? How does platform handle one function in ts and another in js?

Reply:
hmm..this is probably a bad place to place this setting, given it may convey the wrong message. `DefaultLanguage` is only used when creating a new function to select the right template. Edge Runtime itself can handle both ts/js functions without any modifications (need to set via Entrypoint when bundling code).

Alternatively, we can add a top-level config `FunctionsDefaultLanguage`? (can't add it under `Functions` due to possible naming conflicts)

Reply:
discussed offline, will include a comment to clarify